index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta charset="utf-8">
+    <title>Devon Hackley</title>
+    <link rel="stylesheet" href="vendor/styles/normalize.css">
+    <link rel="stylesheet" href="vendor/icons.css">
+    <link rel="stylesheet" href="styles/base.css">
+    <link rel="stylesheet" href="styles/layout.css">
+    <link rel="stylesheet" href="styles/modules.css">
+  </head>
+  <body>
+    <main>
+      <header>
+        <nav id="site-nav">
+            <div class="icon-menu"></div>
+            <ul>
+              <li class=""><a href="#" class="icon-home3"></a>Home</li>
+              <li class=""><a href="#" class="icon-power"></a>About Me</li>
+              <li class=""><a href="https://github.com/devonhackley" class="icon-github"></a>Github</li>
+              <li class=""><a href="#" class="icon-twitter3">Twitter</a></li>
+            </ul>
+        </nav>
+        <h1>Devon's Portfolio</h1>
+      </header>
+      <section class="left-content">
+        <ul id="stuff">
+          <li><img src="http://orig01.deviantart.net/c2b9/f/2016/131/b/e/rowlet_by_vayreceane-da23nxt.jpg"></li>
+        </ul>
+      </section>
+      <section class="center-content">
+        <article class="box">
+          <p class="p-color">
+            Bulbasaur Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ivysaur Lorem ipsum dolor sit amet, consectetur adipiscing elit. Venusaur Lorem ipsum dolor sit amet, consectetur adipiscing elit. Charmander Lorem ipsum
+          </p>
+        </article>
+        <article class="box">
+            <p class="p-color">
+              Bulbasaur Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ivysaur Lorem ipsum dolor sit amet, consectetur adipiscing elit. Venusaur Lorem ipsum dolor sit amet, consectetur adipiscing elit. Charmander Lorem ipsum
```

Comment:
I'd recommend keeping your html clean and removing these `article`s above if you don't absolutely need them.